### PR TITLE
#154953778 How to present deactivated users in gym overview

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
#### What does this PR do?
Decide how to present deactivated users in gym overview

#### Description of Task to be completed?
List only active members in the default list and have a second overview page for the deactivated ones.

#### How should this be manually tested?
When you access the gym users view, you can use the toggle button to see the inactive and active gym users.

#### Any background context you want to provide?
For one to know whether a gym user is active or inactive, they would click on the user to view their status.

#### What are the relevant pivotal tracker stories?

[154953778](https://www.pivotaltracker.com/story/show/154953778)
#### Screenshots (if appropriate)
screen shot 2018-02-14 at 15 27 10
screen shot 2018-02-14 at 15 27 23

### CODE REVIEW

**Comment by hadijahkyampeire** [comment]
@Mercy-Muchai My branch is ready for merging I have solved all the rebase conflicts

**Comment by hadijahkyampeire** [comment]
@Mercy-Muchai I have also made the final squash, you can see it below the comment

**Comment by muchai-mercy** [comment]
Still two commits. Please squash

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,4 @@ node_modules
 venv-django
```
Comment (muchai-mercy) [inline .gitignore]: Do we need to have this and line 65 too?

**Review by muchai-mercy — APPROVED** [review]
LGTM

**Pull request merged**